components/Nav/Nav.tsx

```diff
@@ -56,7 +56,7 @@ export default function Nav({
 
         {showBackButton && (
           <Link href={`/politicians`} passHref>
-            <a aria-label="Go back">
+            <a href={`/politicians`} aria-label="Go back">
```

Comment:
This was needed due to a stupid issue - https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/402